utils/httpSignature.verifiers.ts

```diff
@@ -0,0 +1,60 @@
+import * as crypto from "crypto";
+import { JsonWebKey } from "crypto";
+import {
+  AlgorithmTypes,
+  verifyEcdsaSha256
+} from "@mattrglobal/http-signatures";
+
+// ----------------------
+// Custom Verifiers
+// ----------------------
+
+export const verifyRsaPssSha256 = (key: JsonWebKey) => async (
+  data: Uint8Array,
+  signature: Uint8Array
+): Promise<boolean> => {
+  const keyObject = crypto.createPublicKey({ format: "jwk", key });
+  return crypto
+    .createVerify("SHA256")
+    .update(data)
+    .verify(
+      {
+        dsaEncoding: "ieee-p1363",
+        key: keyObject,
+        padding: crypto.constants.RSA_PKCS1_PSS_PADDING
+      },
+      signature
+    );
+};
+
+export type SupportedAlgTypes = keyof typeof myAlgMap;
+
+const isSupportedAlg = (alg: string): alg is SupportedAlgTypes =>
+  alg === "ecdsa-p256-sha256" || alg === "rsa-pss-sha256";
+
+export const myAlgMap = {
+  ["ecdsa-p256-sha256"]: {
+    verify: verifyEcdsaSha256
+  },
+  ["rsa-pss-sha256"]: {
+    verify: verifyRsaPssSha256
+  }
+};
```

Comment:
Can we extend the `AlgMap` from `@mattrglobal/http-signatures`? Now this implementation support only 2 algos.

Reply:
Yes, I thought about it. Since the app is configured with these two algos, I decided this way.
I'll change it..

Reply:
Done in [b2a775e](https://github.com/pagopa/io-functions-lollipop/pull/30/commits/b2a775e13e7b0043b632e9ac979d2e16d58a6145)